### PR TITLE
Fix #274521: fix concert pitch clef issues

### PR DESCRIPTION
**Pull request by jthistle**
https://musescore.org/en/node/274521

### CODE REVIEW

File: share/instruments/instruments.xml [hunk 1]
```diff
@@ -1066,10 +1066,13 @@
                   <shortName>Hph.</shortName>
                   <description>Heckelphone</description>
                   <musicXMLid>wind.reed.heckelphone</musicXMLid>
-                  <clef>G8vb</clef>
+                  <clef>G</clef>
```
Comment (shoogle) [inline share/instruments/instruments.xml]: `<clef>` should be `<transposingClef>`, and `<concertClef>` should appear immediately afterwards. [See piccolo](https://github.com/musescore/MuseScore/blob/84d4d73dd744b30cedcf2b9e23b1db1708a383c4/share/instruments/instruments.xml#L88-L89) for example.

Comment (jthistle) [reply share/instruments/instruments.xml]: My bad, I'll change that and update the branch

File: share/instruments/instruments.xml [hunk 2]
```diff
@@ -2044,6 +2047,7 @@
                   <pPitchRange>22-60</pPitchRange>
                   <transposeDiatonic>-7</transposeDiatonic>
                   <transposeChromatic>-12</transposeChromatic>
+                  <concertClef>F8vb</concertClef>
```
Comment (shoogle) [inline share/instruments/instruments.xml]: Same problem here as [above](https://github.com/musescore/MuseScore/pull/3824/files#r204242929).

**Comment by shoogle** [comment]
This looks good to me. Thanks for applying my suggestions. Now you just need to squash these commits into one single commit.

### Squashing commits

- **Easy method:** [soft reset and commit](https://stackoverflow.com/a/5201642)
    - This method is fine for squashing multiple commits into one (e.g. 2 into 1, like you are doing here).
    - It also allows you to edit that one squashed commit.
- **Advanced method:** [interactive rebase](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)
    - This method can be used to squash commits (e.g. 2 into 1) and it also allows you to do more fancy things like split commits (e.g. 1 into 2), or squash into more than one (e.g. 3 into 2).
    - It also allows you to edit commits earlier than the most recent one.

You can use the easy method. The advanced method is more complicated and is only useful for things that the easy method can't handle. I only mention the advanced method because for some reason it is usually given as the method for squashing commits, even though the easy method works fine for this.

**Comment by jthistle** [comment]
Squashed and ready for merge :)

**Comment by lasconic** [comment]
If I'm not wrong, this is your first PR to MuseScore ! Thank you and welcome to the contributor community !

**Comment by jthistle** [comment]
@lasconic - it certainly is! I'm looking forward to making many more, I hope :)

**Pull request merged**